packages/@storefront/core/tsconfig.esnext.json

```diff
@@ -6,7 +6,9 @@
       "DOM"
     ],
     "types": [
-      "node"
+      "node",
```

Comment:
You shouldn't have to change the `esnext` tsconfigs :s

Reply:
Change introduce as part of #10. Marking as resolved.